README.md

```diff
@@ -11,99 +11,205 @@
 
 Join the Data API builder community! This sign up will help us maintain a list of interested developers to be part of our roadmap and to help us better understand the different ways DAB is being used. Sign up [here](https://forms.office.com/pages/responsepage.aspx?id=v4j5cvGGr0GRqy180BHbR1S1JdzGAxhDrefV-tBYtwZUNE1RWVo0SUVMTkRESUZLMVVOS0wwUFNVRy4u).
 
-## About
-
-**Data API builder for Azure Databases provides modern REST and GraphQL endpoints to your Azure Databases.**
-
-With data API builder, database objects can be exposed via REST or GraphQL endpoints so that your data can be accessed using modern techniques on any platform, any language, and any device. With an integrated and flexible policy engine, native support for common behavior like pagination, filtering, projection and sorting, the creation of CRUD backend services can be done in minutes instead of hours or days, giving developers an efficiency boost like never seen before.
-
-Data API builder is Open Source and works on any platform. It can be executed on-premises, in a container or as a Managed Service in Azure, via the new [Database Connection](https://learn.microsoft.com/azure/static-web-apps/database-overview) feature available in Azure Static Web Apps.
-
-![Data API builder Architecture Overview Diagram](./docs/media/dab-architecture-overview.png)
-
-## Features
-
-- Allow collections, tables, views and stored procedures to be accessed via REST and GraphQL
-- Support authentication via OAuth2/JWT
-- Support for EasyAuth when running in Azure
-- Role-based authorization using received claims
-- Item-level security via policy expressions
-- REST
-  - CRUD operations via POST, GET, PUT, PATCH, DELETE
-  - Filtering, sorting and pagination
-- GraphQL
-  - Queries and mutations
-  - Filtering, sorting and pagination
-  - Relationship navigation
-- Easy development via dedicated CLI
-- Full integration with Static Web Apps via Database Connection feature when running in Azure
-- Open Source
+![](docs/media/dab-logo.png)
+
+## About Data API builder
+
+Data API builder (DAB) is an open-source, no-code tool that creates secure, full-featured REST and GraphQL endpoints for your database. It’s a CRUD data API engine that runs in a container—on Azure, any other cloud, or on-premises. DAB is built for developers with integrated tooling, telemetry, and other productivity features.
+
+```mermaid
+erDiagram
+    DATA_API_BUILDER ||--|{ DATA_API : "Provides"
+    DATA_API_BUILDER {
+        container true "Microsoft Container Repository"
```

Comment:
What does the column containing only true values mean?

Reply:
I think true here just means DAB provides that feature